### ISSUE TITLE
Mobile - Dropdown Menu Keep Focused

### ISSUE THREAD

**Issue by marioaleogolsat** [angular/material#4326]
I'm facing an awkward issue, i'm working on a project using ngMaterial with Ionic and i`ve made a dropdown menu. 

```
<md-toolbar layout="row" layout-align="start center" ng-controller="appCtrl" ng-show="isActive()">
  <span>
    <md-menu>
      <md-button class="md-icon-button" arial-label="Menu" ng-click="$mdOpenMenu($event)">
              <md-icon md-svg-icon="img/ic_list_24px.svg"></md-icon>
      </md-button>
      <md-menu-content width="3">      
        <md-menu-item><md-button href="#/tab/dash" ng-click="$mdCloseMenu($event)">Dashboard</md-button></md-menu-item>

        <md-menu-item><md-button href="#/tab/chats" ng-click="$mdCloseMenu($event)">Chats</md-button></md-menu-item>

        <md-menu-item><md-button ng-click="$mdMenuHide()" href="#/tab/account" >Account</md-button></md-menu-item>
      </md-menu-content>
    </md-menu>
  </span>

  <span>
    <h3><span id="titlebar">{{currentTitle()}}</span></h3>
  </span>
</md-toolbar>
```

When I click on the menu and then click anywhere it closes the menu and the button stay unfocused, ok, it's working fine, but when I click on the menu and then I select an option and it close the menu and change the url the menu stay focused until I click on it and then click somewhere else. 
Is there a way to set of the focus option, once I'm working on mobile and "it' s irrelevant".

**Comment by rschmukler** [comment]
Could be related to #4402 and menus not cleaning up on a `$scope.$destroy`. Needs review.

**Comment by rschmukler** [comment]
I suspect this was resolved by #4402. @marioaleogolsat if the issue persists please open a new issue and reference this issue in it. Thanks!
